### PR TITLE
Heads' door remotes and access tuners now have a radial menu

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Heads' door remotes and access tuners now have a radial menu.

## Why It's Good For The Game
Quality of life, makes the horrendous door remotes suck less

## Images of changes

Heres what you see when you open the radial, and the door menu is the previously selected option.
image
dreamseeker_7IL2LlD0Ut

## Testing

Access tuners, and head remotes still work as intended, being able to change between all modes and use them. Doesn't affect the janitor's keyring

## Changelog
:cl:
tweak: Door remotes and access tuners now use radial menus, for ease of use.
/:cl:

### CODE REVIEW

File: code/game/objects/items/control_wand.dm [hunk 1]
```diff
@@ -122,7 +124,7 @@
 	region_access = list(REGION_MEDBAY)
 
 /obj/item/door_remote/civillian
-	name = "civilian door remote"
+	name = "civillian door remote"
```
Comment (Henri215) [inline code/game/objects/items/control_wand.dm:127]:
```suggestion
	name = "civilian door remote"
```

File: code/game/objects/items/control_wand.dm [hunk 2]
```diff
@@ -138,7 +140,7 @@
 	desc = "A device used for illegally interfacing with doors."
 	icon_state = "hacktool"
 	item_state = "hacktool"
-	var/hack_speed = 1.5 SECONDS
+	var/hack_speed = 30
```
Comment (Henri215) [inline code/game/objects/items/control_wand.dm:143]:
```suggestion
	var/hack_speed = 1.5 SECONDS
```

**Review by Henri215 — COMMENTED** [review]
Looks like you ended up reverting these changes by accident?

File: code/game/objects/items/control_wand.dm (hunk 1 above)
Comment (Contrabang) [reply code/game/objects/items/control_wand.dm:127]: epic old master fail

**Review by Burzah — APPROVED** [review]

**Pull request merged**